docs/CSDK-guide.md

```diff
@@ -126,12 +126,39 @@ bool readFromS3(JNIEnv *env, char *argv[]) {
     reader.close();
 }
 
-// 3. destory JVM
+// 3. read schema
+/**
```

Comment:
This can be removed.

As after #2792.  we just put link to main.cpp. No need to duplicate the code

Reply:
yes, removed